reactive/docker_registry.py

```diff
@@ -246,13 +246,20 @@ def update_reverseproxy_config():
     #  https://bugs.launchpad.net/layer-docker-registry/+bug/1815459
     common_opts = "check inter 2000 rise 2 fall 5 maxconn 4096"
     is_primary = True
+    tls_ops = ""
+    if (
+        is_flag_set('config.set.tls-cert-path') and
+        is_flag_set('config.set.tls-key-path')
+    ):
+        tls_ops = "ssl verify none"
```

Comment:
I think we want `ssl verify required` here.  We've let users know valid certs are required in this setup, and so ssl health checks should be used/work.  We'd want `none` for "SSL passthru", but we're not supporting that here.